src/zeusops_bot/reforger_config_gen.py

```diff
@@ -55,13 +70,14 @@ def zeus_upload(
             base_config_content = json.loads(self.base_config.read_text())
         except json.JSONDecodeError as e:
             raise errors.ConfigFileInvalidJson(e)
-        modded_config_dict = patch_file(base_config_content, modlist, scenario_id)
+        mods = extract_mods(modlist)
```

Comment:
As mentioned above, strongly recommend to move this call OUT to the calling site (cogs/zeus_upload.py)

Reply:
Same as above, unintentional